### DISCUSSION TITLE
Casting via DLNA/UPnP

### DISCUSSION THREAD

**Discussion by TechD123** [Radiokot/photoprism-android-client#111] [Ideas]
Aves has recently added [support for casting via DLNA/UPnP](https://github.com/deckerst/aves/blob/develop/CHANGELOG.md#v1.10.0) and I thought it might be something worth considering. This would allow users to share their content to the big screen (e.g. TV with Kodi or DLNA/UPnP out of the box).

Thanks for this great app, I'd love to donate! Could you add Stripe to your liberapay profile?

**Comment by Radiokot** [comment]
Hi. Thanks for suggestion, DLNA support is in the app backlog already.
I'm glad that you want to make a donation to support my work. Please, use one of available methods on my site: https://radiokot.com.ua/tip
I can't use Stripe.

**Reply by TechD123** [reply]
Don't have a credit card or PayPal account unfortunately. Guess I'll wait until the Ukraine joins the EU ;)

**Reply by Radiokot** [reply]
Don't wait, join Bitcoin 😉
